test/fixtures.js

```diff
@@ -857,5 +857,73 @@ return [
             ]
         },
 
+    // POST REPORT - XENOPHOBIA, LOOTING
+        {
+            'repeatable': true,
+            'request': {
+                'method': 'POST',
+                'headers': {
+                    'Authorization': ['Basic ' + new Buffer('test:test').toString('base64')],
+                    'Content-Type': ['application/json']
+                },
+                'url': 'https://nightingale_root/api/v1/report/',
+                "data": {
+                    "contact_key": "contact_key_064001",
+                    "to_addr": "+064001",
+                    "categories": ["111","444"],
+                    "location": {
+                        "point": {
+                            "type": "Point",
+                            "coordinates": ["3.1415","2.7182"]
+                        }
+                    },
+                    "metadata": {
+                        "language": "fr",
+                        "status": "refugee",
+                        "country": "drc"
+                    },
+                    "description":""
+                }
+            },
+            'response': {
+                "code": "200",
+                "data": []
+            }
+        },
+
+    // POST REPORT - CORRUPTION, OTHER
+        {
+            'repeatable': true,
+            'request': {
+                'method': 'POST',
+                'headers': {
+                    'Authorization': ['Basic ' + new Buffer('test:test').toString('base64')],
```

Comment:
I know this isn't checked by harness at the moment, but for consistency sake, can you make this reflect reality of Token auth?